packages/cli/src/index.ts

```diff
@@ -128,7 +128,7 @@ yargs
   )
 
   .command(
-    'push [api] [maybeDestination] [maybeBranchName]',
```

Comment:
Yargs will not parse positional arguments without it. We need it for backward compatibility.

Reply:
Ah, got it - and fixed. Thanks!